### PR TITLE
`subliminal-test` can now be run alongside the Xcode 5 toolchain, and respects the "workspace" argument.

### PR DESCRIPTION
**Pull request by wearhere**

### CODE REVIEW

**Comment by wearhere** [comment]
This fixes both #31 and #33 because I couldn't test the former without the latter.

**Comment by wearhere** [comment]
For anyone keeping score, the attempt to convert `reset_simulator` into a text document (9f54bc83dc6d110dbe840cda5a35cbab60542d2b) [failed](https://travis-ci.org/inkling/Subliminal/builds/8333273#L1104) and so was removed. :cry:

**Comment by wearhere** [comment]
Bloody hell, `reset_simulator` timed out again even as a compiled script. I'll look into replacing it with a programmatic solution tomorrow.

**Comment by HaloZero** [comment]
@wearhere where was the script timing out? I felt like adding a delay between application launch and trying to set the menu option would work.

File: README.md [hunk 1]
```diff
@@ -277,9 +277,22 @@ application and tests.
...
+Master, but there is at least [one serious bug](http://openradar.appspot.com/radar?id=3115412) 
+in the current 5.0 developer tools that would need to be resolved by Apple before we 
+could fully support that toolchain.
+
+If you have Xcode 5 installed, your [continuous integration script](#continuous-integration) 
+must select the Xcode 4.6 toolchain before invoking `subliminal-test`:
+
+```sh
+export DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer
+```
```
Comment (MaxGabriel) [inline README.md:295]: Is this different from `xcode-select`?

Comment (wearhere) [inline README.md:295]: It's documented in `xcode-select`'s `man` page as having the same effect as `xcode-select --switch <path>`--but only within the same shell & subshells, so devs can use the Xcode 5 toolchain normally if they like. The other benefit vs. `xcode-select --switch` is that setting `DEVELOPER_DIR` doesn't require superuser privileges.

**Comment by MaxGabriel** [comment]
Yep, this works perfectly:

[x] On a workspace with only Xcode 4.6 installed
[x] On a workspace with both Xcode 4.6 and Xcode 5 installed
[x] On a regular project, with both Xcode 4.6 and Xcode 5 installed

**Comment by wearhere** [comment]
@HaloZero It's difficult to say where the script was timing out because Applescript errors [don't include line-numbers](https://travis-ci.org/inkling/Subliminal/builds/8333273#L1104) (at least not in a form which corresponds to the decompiled source). Your guess is reasonable. I don't think that it's worth debugging when we can reset the content and settings directly (fbb4ddf), though--I'd rather not worry about timing issues across different environments (to say nothing of the fragility of trying to script applications' interfaces anyway).

@MaxGabriel Thanks for running those tests. @inkling/committers, this is ready for review.

**Comment by wearhere** [comment]
Lol that's not the right group. And it appears that I don't have permissions to create a more appropriate one... ok how about @ahaneyinkling or @aegolden, one of you please.

**Pull request merged**